README.md

```diff
@@ -51,6 +51,29 @@ const order = await rarible.createOrder({
 
 console.log(order); // {..., hash: "abc123"}
 
+
+// Update that sell order to do a "flash sale":
+//  - Lower price from 1.75 ETH to 1.5 ETH
+//  - Sell order will now expire in 24 hours
+
+order = await rarible.updateOrder({          
+    //**
+    //* Just passing an order hash will query fill in all necessary Order data
+    // **
+    hash: "abc123",  
+
+    //**
+    //* Additional parameters given to the method will be updated on the order. 
+    //* (This function needs a lot of error handling, no updating someone else's order, can only lower price, order is still unmatched, etc)
+    //** 
+    amount: "1500000000000000000",
+    endBlockTimestamp: 120324374
```

Comment:
Why are we passing the end block timestamp? It sounds like the only thing that can be updated on an order is its amount at the moment (which can only be lowered).

Reply:
You're right @benefacto, I added some changes to this PR, please give another look

Reply:
Looks great! I think just the readme needs to be updated with your changes.